isp/bind/named.conf.local

```diff
@@ -0,0 +1,40 @@
+//
+// Do any local configuration here
+//
+
+// Consider adding the 1918 zones here, if they are not used in your
+// organization
+//include "/etc/bind/zones.rfc1918";
+
+view "internal" {
```

Comment:
Create internal zone for customers inside our network and allow them to query corporate domains and services.